src/app/codelabs/angular/create-first-app/create-first-app.component.html

```diff
@@ -253,14 +253,61 @@ <h2 i18n>Like a component, the <b>Angular</b> module is adorned with a decorator
       /
       <ng-container i18n>Browser Module</ng-container>
     </h1>
-    <h2 i18n>Because we're building a browser web app, we need to pass <b>BrowserModule</b> to the <b>imports</b> array
-    </h2>
-    <slides-code-editor path="app.module.ts" [fontSize]="30" [code]="code.moduleAnatomy.code"
-                        [focus-highlight-match]="[code.moduleAnatomy.matches.importsArr]"></slides-code-editor>
-
-    <div class="info" i18n>
-      Angular is not just for web apps anymore; you can also use it to create native phone apps and even
-      VR scenes.
+    <ng-container *ngIf="currentMode=='web'">
+        <h2 i18n>Because we're building a browser web app, we need to pass <b>BrowserModule</b> to the <b>imports</b> array</h2>
+    </ng-container>
+    <ng-container *ngIf="currentMode=='mobile'">
+        <h2 i18n>With Angular we build mobile apps using NativeScript or Ionic.</h2>
+    </ng-container>
+    <ng-container *ngIf="currentMode=='vr'">
```

Comment:
I think ngSwitch would be better for these cases. 
https://angular.io/api/common/NgSwitch